src/main/java/calculator/domain/Operation.java

```diff
@@ -0,0 +1,5 @@
+package calculator.domain;
+
+public interface Operation {
+    int calculate(int a, int b);
```

Comment:
```suggestion
    int calculate(int operand1, int operand2);
```
a, b 보다는
의미있는 이름으로 정의하는 것이 좋겠어요 😄

num1, num2도 좋지만
Operation에 맞게 추천드려봅니다.